src/Writer/CsvWriterTerminate.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
```

Comment:
This should have been `Sonata\Exporter\Writer`

Reply:
Nop, my bad, after checking stuff, I saw that on 1.x we still have `Exporter\....` and on master we use `Sonata\Exporter`...

Reply:
Careful with the automated merge then